app/views/hub/coalitions/_form.html.erb

```diff
@@ -0,0 +1,26 @@
+<div>
+  <%= form_with model: @coalition, url: url, method: local_assigns[:http_method], local: true, builder: VitaMinFormBuilder, id: "coalition-form" do |f| %>
+    <h1 class="form-card__title">
+      <%= @title %>
+    </h1>
+
+    <%= f.cfa_input_field :name, t("general.name") %>
+      <div class="form-group state-select">
+        <label for="state-select" class="form-question"><%= t("general.state") %></label>
+        <div>
+          <%= text_field_tag("state_routing_targets[states]",
+                             @coalition.state_routing_targets.map { |srt| States.name_for_key(srt.state_abbreviation) }.to_json.html_safe,
+                             id: "state-select", class: "text-input multi-select-state") %>
+        </div>
+      </div>
+    <div>
+      <%= f.submit t("general.save"), class: "button button--primary button--wide" %>
+    </div>
+  <% end %>
+</div>
+
+<% content_for :script do %>
+  <script>
+      window.taggableItems = <%= States.html_safe_names %>;
```

Comment:
I'm now seeing that it may be the case that we might be doing the name conversion back and forth because its possibly simpler to display the already selected states properly when reloading ... 😬 

I still think if it's possible to deal with displaying the full name but passing the abbreviation around as values, that would be nice.